packages/vote/fpc/fpc.go

```diff
@@ -321,3 +315,32 @@ func (f *FPC) voteContextIDs() (conflictIDs []string, timestampIDs []string) {
 	}
 	return conflictIDs, timestampIDs
 }
+
+// get round boundaries based on the voting stage
+func (f *FPC) setThreshold(voteCtx *vote.Context) (float64, float64) {
+	lowerThreshold := f.paras.SubsequentRoundsLowerBoundThreshold
+	upperThreshold := f.paras.SubsequentRoundsUpperBoundThreshold
+
+	if voteCtx.HadFirstRound() {
+		lowerThreshold = f.paras.FirstRoundLowerBoundThreshold
+		upperThreshold = f.paras.FirstRoundUpperBoundThreshold
+	}
+
+	if voteCtx.HadFixedRound(f.paras.CoolingOffPeriod, f.paras.FinalizationThreshold, f.paras.FixedEndingThreshold) {
```

Comment:
can these paras be passed with f alone or is there an advantage of sending them individually? 
or does this make the setup for the test too complicated?

Reply:
a little easier setup for tests